### PR TITLE
accordion user menu

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

**Review by elizzhao — APPROVED** [review]

File: src/components/TopBar.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
-import { Fragment } from 'react'
+import { Collapse } from 'react-collapse'
```
Comment (MattPutnam) [inline src/components/TopBar.js]: We also have a src/components/Collapse, which could get confusing.

**Review by MattPutnam — COMMENTED** [review]
I think the alternate background gray for the profile row is a little harsh, and I think that the Profile/Groups/Sign Out items should have the same size and horizontal spacing as the items below them (being shorter is fine by me).

**Comment by MattPutnam** [comment]
I'm also getting a console error about a missing prop

File: src/components/TopBar.js (hunk 1 above)
Comment (zarsky-broad) [reply src/components/TopBar.js]: Ooh, I forgot about that. Let me combine these.

**Comment by zarsky-broad** [comment]
@MattPutnam re-review, plz

File: src/libs/colors.js [hunk 2]
```diff
@@ -63,5 +63,6 @@ export default {
   ],
 
   slate: '#8399a4',
-  brick: '#c35b69'
+  brick: '#c35b69',
+  lighten: 'rgba(255, 255, 255, 0.2)'
```
Comment (MattPutnam) [inline src/libs/colors.js]: I don't like having this one singular alpha color filter. I'd like to see the places that use this declare their colors absolutely.

**Review by MattPutnam — APPROVED** [review]
👍, one small concern.

**Pull request merged**